src/App.tsx

```diff
@@ -12,6 +13,7 @@ function App() {
         <Routes>
           <Route path="/" element = {<HomePage />}></Route>
         </Routes>
+        <Contact/>
```

Comment:
if Contact is a part of HomePage then it means the tag for it must be added inside of HomePage, but not inside of App.tsx =)